druid-derive/src/lib.rs

```diff
@@ -24,7 +24,7 @@ use proc_macro::TokenStream;
 use syn::parse_macro_input;
 
 /// Generates implementations of the `Data` trait.
-#[proc_macro_derive(Data, attributes(druid))]
```

Comment:
is there a way we can keep `druid` attribute, and print a helpful error if we see it?

Reply:
We still printing error without it. This will additionally show error that attribute is not supported. Would you like to add it here to expected attributes, and only show error during parsing?

Reply:
In a perfect world we could print a compiler warning but still function if `druid` was used. I think this is maybe hard to do, though, so I'm happy with this how it is.